app/services/update_duplicate_matches.rb

```diff
@@ -30,7 +30,14 @@ def save_match(match)
   end
 
   def create_or_update_fraud_match(match)
-    fraud_match = FraudMatch.find_by(last_name: match['last_name'], postcode: match['postcode'], date_of_birth: match['date_of_birth'])
+    fraud_match = FraudMatch.where(
+      'TRIM(UPPER(last_name)) = ?',
+      match['last_name'].upcase.strip,
+    ).where(
+      "REPLACE(UPPER(postcode), ' ', '') = ?",
+      match['postcode'].upcase.gsub(' ', ''),
+    ).where(date_of_birth: match['date_of_birth']).first
```

Comment:
This is good. Could we arrange for all the routines to use the same query?

Reply:
Good question. I think the other place where this is needed is the old `UpdateFraudMatches` class that will be replaced with this one. Have extracted this into a separate helper method on the model. There is still duplication with this and the SQL query in `GetFraudMatches` - not sure there's an easy way to solve that.